ref_builder/otu/update.py

```diff
@@ -69,8 +70,23 @@ def add_isolate(
             f"The schema requires {len(otu.schema.segments)} segments: "
             + f"{[segment.name for segment in otu.schema.segments]}"
         )
+
         return
 
+    if (isolate_id := otu.get_isolate_id_by_name(isolate_name)) is not None:
+        otu_logger.debug(f"{isolate_name} already exists in this OTU")
+
+        for record in isolate_records.values():
+            if record.accession.startswith("NC_"):
+                otu_logger.debug("Accession is RefSeq")
+                update_isolate_from_records(repo, otu, isolate_id, list(isolate_records.values()))
+
+                otu = repo.get_otu(otu.id)
+
+                return otu.get_isolate(isolate_id)
+
+            return None
```

Comment:
If I understand, this is going to update an isolate with refseq accessions if it doesn't already have them. This is more than `add_isolate` should do.

I am not reading too deep, so I may be missing something.

Reply:
Yes, it keeps `add_isolate()` from failing automatically if the isolate name is the same but records are from RefSeq. It is a pretty sloppy check, though.

Reply:
Okay, we'll handle this in a future PR.